setup.py

```diff
@@ -21,5 +24,8 @@
           'scipy',
           'scikit-learn'
       ],
-      test_suite='test'
+      test_suite='test',
+      keywords=['Metric Learning', 'Large Margin Nearest Neighbor', \
```

Comment:
The line continuations aren't necessary here. I would format this like the list of `install_requires`, with one per line.